CHANGELOG.rst

```diff
@@ -2,6 +2,16 @@
 CHANGELOG
 =========
 
+2.7.1
+==========
+* improvement: feat(sqla-core): Add support for rendering Database Specific queries. `PR291 <https://github.com/aws/aws-xray-sdk-python/pull/291>`_.
+* bugfix: Fixing broken instrumentation for sqlalchemy >= 1.4.0. `PR289 <https://github.com/aws/aws-xray-sdk-python/pull/289>`_.
+* improvement: no op trace id generation. `PR293 <https://github.com/aws/aws-xray-sdk-python/pull/293>`_.
+* bugfix: Handle exception when sending entity to Daemon. `PR292 <https://github.com/aws/aws-xray-sdk-python/pull/292>`_.
+* bugfix: Fixed serialization issue when cause is a string. `PR284 <https://github.com/aws/aws-xray-sdk-python/pull/284>`_.
+* improvement: Run tests against Python 3.9 and Django 3.2 LTS. `PR285 <https://github.com/aws/aws-xray-sdk-python/pull/285>`_.
```

Comment:
Ideally changelog should only contain customer related changes. Operational stuff like this and the availability metrics are for us developers and can be skipped from the changelog.